.jenkins

```diff
@@ -1,16 +1,39 @@
+def getSafeBranchName() {
+	return "${env.BRANCH_NAME}".replace('/', '-')
+}
+
+def getTagName() {
+	def branchName = getSafeBranchName()
+	if ("${env.BRANCH_NAME}" == 'development') {
```

Comment:
We can drop the build stuff, that was only for the KE portal. Instead we use the `develop` and `stable` branches for code promotion